tests/indexes/tests.py

```diff
@@ -72,14 +75,16 @@ def test_index_together(self):
         # Ensure the index name is properly quoted
         self.assertIn(
             connection.ops.quote_name(
-                editor._create_index_name(
-                    Article._meta.db_table, ["headline", "pub_date"], suffix="_idx"
-                )
+                str(Article._meta.indexes[0].create_sql(Article, editor))
             ),
-            index_sql[0],
+            connection.ops.quote_name(index_sql[0]),
```

Comment:
This tests should check that the index names is properly quoted, it doesn't check it anymore because we quote the entire `CREATE INDEX` statement. This should fail with:
```diff
diff --git a/django/db/backends/base/schema.py b/django/db/backends/base/schema.py
index f2ca8c8df9..842b0e977f 100644
--- a/django/db/backends/base/schema.py
+++ b/django/db/backends/base/schema.py
@@ -1348,7 +1348,7 @@ class BaseDatabaseSchemaEditor:
             nonlocal name
             if name is None:
                 name = self._create_index_name(*args, **kwargs)
-            return self.quote_name(name)
+            return name
 
         return Statement(
             sql_create_index,
```

Reply:
We should also rename this tests, it's called `test_index_together`.

Reply:
Indeed, I misunderstood the test case :)